backend/config/curation-api.yml

```diff
@@ -1003,7 +1003,6 @@ components:
         | `"VALIDATION_FAILURE"` | An error occured during validation. See `processing_status_detail`|
         | `"PIPELINE_FAILURE"`| An error occured during processing. |
       enum:
-        - FAILURE
         - PENDING
         - SUCCESS
         - VALIDATION_FAILURE
```

Comment:
why are these two lines not showing up as additions in the diff... isn't this what you are purportedly adding? 🤔